src/parser/deathknight/frost/modules/features/BreathOfSindragosa.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS';
+import SpellIcon from 'common/SpellIcon';
+import SpellLink from 'common/SpellLink';
+
+import StatisticBox, { STATISTIC_ORDER } from 'interface/others/StatisticBox';
+
+import Analyzer from 'parser/core/Analyzer';
+
+class BreathOfSindragosa extends Analyzer{
+  static dependancies = {
+
+  }
+
+  beginTimestamp = 0;
+  casts = 0;
+  badCasts = 0;
+  totalDuration = 0;
+  breathActive = false;
+
+  on_byPlayer_applybuff(event){
+    const spellId = event.ability.guid;
+    if(spellId !== SPELLS.BREATH_OF_SINDRAGOSA_TALENT.id){
+      return;
+    }
+    this.casts += 1;
+    this.beginTimestamp = event.timestamp;
+    this.breathActive = true;
+  }
+
+  on_byPlayer_removebuff(event){
+    const spellId = event.ability.guid;
+    if(spellId !== SPELLS.BREATH_OF_SINDRAGOSA_TALENT.id){
+      return;
+    }
+    this.breathActive = false;
+    const duration = event.timestamp - this.beginTimestamp;
+    if(duration < 15000){
```

Comment:
I would suggest making this a const `GOOD_CAST_THRESHOLD_MS` or something like that.